test/release/acorn.sh

```diff
@@ -4,13 +4,14 @@ alias uglify-js=$PWD/bin/uglifyjs
 UGLIFY_OPTIONS=$@
 
 minify_in_situ() {
+    ARGS="$UGLIFY_OPTIONS --in-situ"
     DIRS="$1"
     echo '> uglify-js' $DIRS $UGLIFY_OPTIONS
     for i in `find $DIRS -name '*.js'`
     do
-        echo "$i"
-        uglify-js "$i" $UGLIFY_OPTIONS -o "$i"
+        ARGS="$ARGS $i"
```

Comment:
An advantage of the for/do loop is you could deal with whitespace in paths via:
```
         ARGS="$ARGS '$i'"
```

Reply:
Hold on... this could work:
```
$ ARGS="-mc --in-situ"; for i in `find 'f o o' 'b a r' -type f -name '*js' -exec echo "'{}'" \;` ; do ARGS="$ARGS $i"; done; CMD="bin/uglifyjs $ARGS"; echo $CMD ; eval $CMD
bin/uglifyjs -mc --in-situ 'f o o/0.js' 'f o o/4.js' 'f o o/5.mjs' 'b a r/2.js' 'b a r/7.js'
f o o/0.js
f o o/4.js
f o o/5.mjs
b a r/2.js
b a r/7.js
```
Shell scripting for fun and profit!

Reply:
Ooooh - find grouping operator:
```
$ ARGS="-mc --in-situ"; for i in `find 'f o o' 'b a r' -type f \( -name '*.js' -o -name '*.mjs' \) -exec echo "'{}'" \;` ; do ARGS="$ARGS $i"; done; CMD="bin/uglifyjs $ARGS"; echo $CMD ; eval $CMD
bin/uglifyjs -mc --in-situ 'f o o/0.js' 'f o o/4.js' 'f o o/5.mjs' 'b a r/2.js' 'b a r/7.js'
f o o/0.js
f o o/4.js
f o o/5.mjs
b a r/2.js
b a r/7.js
```

Reply:
... what about `'` in file names? 🤔

Reply:
> ... what about ' in file names?

The gods of unix shall cast a curse upon any person who dares use a quote in a filename.

I've always wanted to use the shell `$IFS` - who knows - maybe it is applicable here: https://bash.cyberciti.biz/guide/$IFS

Reply:
To think that people (still) laugh at 8.3 of Microsoft that hard after all these years... 😏

Reply:
Okay I've got this concoction:
```sh
$ find 'f o o' 'b a r' -name '*js' -exec sh -c 'printf "%q\n" "$@"' printf {} \;
f\ o\ o/0.js
f\ o\ o/4.js
f\ o\ o/5\'.mjs
b\ a\ r/\".js
```
I'll see myself out.

Reply:
That's good - but then you've got to get `bin/uglifyjs` to handle that output somehow.

Reply:
Oh I expect it to work with `for` and construct a normal space-delimited argument list from there, since the other spaces are `\`-escaped properly so it shouldn't confuse the CLI anymore... 🤔

Once I can get past my PTSD may be I'll commit this with the next build testing PR.